components/PlantCard/index.js

```diff
@@ -49,6 +54,11 @@ const StyledImageContainer = styled.div`
   box-shadow: 0 0px 15px rgba(0, 0, 0, 0.3);
   border-radius: 20px;
 
+  &:hover {
+    box-shadow: 0px 5px 40px -10px var(--white-dark);
+    transition: all 0.45s ease 0s;
+  }
+
   @media (max-width: 750px) {
     width: 170px;
   }
```

Comment:
The @media query for the width of the cards in small screens is good, but could be supplemented to be more consistent. You coul add height to be save that the card looks good.

@media (max-width: 750px) {
  width: 170px;
  height: auto;

Reply:
Thank you for the feedback. This has already been considered, but we plan to adress it in another US focussed on design refactoring and improvement. We believe it is out of scope for this US.